content/zh-cn/docs/reference/glossary/qos-class.md

```diff
@@ -2,9 +2,9 @@
 title: QoS 类（QoS Class）
 id: qos-class
 date: 2019-04-15
-full_link: 
+full_link: /docs/concepts/workloads/pods/pod-qos/
```

Comment:
Since it does not have a zh-cn page yet, I just left it as it is to avoid invalid link.
But I just found that windsonsea is already localizing it (https://github.com/kubernetes/website/pull/40271), so fix as suggested, thanks.